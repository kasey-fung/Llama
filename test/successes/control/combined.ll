int main(int a, int b, int c, int d):
    a = -6
    b = 0
    c = 10
    d = 20

    while(a < c):
        if (a > d):
            print(b)
        elif(a < b):
            print(a)
        a += 2
    
    for(a = 0; a < 3; ++a):
        if (c < d):
            print(c)
        else:
            print(d)
    
    if(b < d):
        a = 0
        while(a < 4):
            print(a)
            a += 2
    else:
        print(b)
    
    if(c < d):
        for(b = 0; b < 4; ++b):
            if(d < 20):
                print(d)
            elif(d == 20):
                print(b)
            elif(d > 20):
                print(d)
    else:
        print(a)
