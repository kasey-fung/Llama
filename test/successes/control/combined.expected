-6
-4
-2
10
10
10
0
2
0
1
2
3
